apps/ui/src/helpers/utils.ts

```diff
@@ -735,5 +765,9 @@ export async function getSpaceController(id: string, network: NetworkID) {
     return owner || EVM_EMPTY_ADDRESS;
   }
 
+  if (id.endsWith('.sonic')) {
+    return getUnstoppableDomainsNameOwner(id, chainMapping.sonic[network]);
+  }
```

Comment:
I wonder, why we used Stamp for `.shib` but contract call on UI directly for `.sonic`?